tsl/test/expected/continuous_aggs_dump.out

```diff
@@ -101,7 +101,7 @@ NOTICE:  adding index _materialized_hypertable_4_location_bucket_idx ON _timesca
 SET client_min_messages TO LOG;
 SET timescaledb.current_timestamp_mock = '2019-02-01 00:00';
 REFRESH MATERIALIZED VIEW mat_before;
-LOG:  materializing continuous aggregate public.mat_before: nothing to invalidate, new range up to Sun Feb 24 16:00:00 2019 PST
+LOG:  materializing continuous aggregate public.mat_before: nothing to invalidate, new range up to Sun Feb 03 16:00:00 2019 PST
 SELECT count(*) FROM conditions_before;
```

Comment:
It is not clear why these changes are in the test. You did a select of the max in a later test, which makes it clear why it is lower. Can you do something similar here?